src/dags/customnodesession.hpp

```diff
@@ -17,9 +17,21 @@
 
 #include <memory>
 #include <string>
+#include <unordered_map>
+#include <utility>
 
 #include <openvino/openvino.hpp>
 
+#include "../custom_node_interface.h"  // NOLINT
+#include "../logging.hpp"
+#include "../profiler.hpp"
+#include "../status.hpp"
```

Comment:
Unfortunately we need to include all those in the header. That's why I'm asking if we really need to move ::execute to header

Reply:
Yes this is essential, unfortunately.